src/components/v5/common/ActionSidebar/partials/ColonyDetailsFields/ColonyDetailsFields.tsx

```diff
@@ -9,82 +9,94 @@ import ActionFormRow from '~v5/common/ActionFormRow/index.ts';
 
+import { useHasNoDecisionMethods } from '../../hooks/index.ts';
+
 import ColonyAvatarField from './partials/ColonyAvatarField/index.ts';
 
 const displayName = 'v5.common.ActionsContent.partials.ColonyDetailsFields';
 
-const ColonyDetailsFields: FC = () => (
-  <>
-    <ActionFormRow
-      icon="pencil-circle"
-      fieldName="colonyName"
-      title={formatText({ id: 'actionSidebar.colonyName' })}
-      tooltips={{
-        label: {
-          tooltipContent: formatText({
-            id: 'actionSidebar.tooltip.editColony.colonyName',
-          }),
-        },
-      }}
-    >
-      <FormInputBase
-        message={false}
-        name="colonyName"
-        maxLength={MAX_COLONY_DISPLAY_NAME}
-        placeholder={formatText({
-          id: 'actionSidebar.colonyName.placeholder',
-        })}
-        mode="secondary"
-      />
-    </ActionFormRow>
-    <ActionFormRow
-      icon="image"
-      fieldName="avatar"
-      title={formatText({ id: 'actionSidebar.colonyLogo' })}
-      tooltips={{
-        label: {
-          tooltipContent: formatText({
-            id: 'actionSidebar.tooltip.editColony.colonyLogo',
-          }),
-        },
-      }}
-    >
-      <div className="flex items-center">
-        <ColonyAvatarField
-          name="avatar"
-          fileOptions={{
-            fileFormat: ['.PNG', '.JPG', '.SVG'],
-            fileDimension: '250x250px',
-            fileSize: '1MB',
-          }}
+const ColonyDetailsFields: FC = () => {
+  const hasNoDecisionMethods = useHasNoDecisionMethods();
+
+  return (
+    <>
+      <ActionFormRow
+        icon="pencil-circle"
+        fieldName="colonyName"
+        title={formatText({ id: 'actionSidebar.colonyName' })}
+        tooltips={{
+          label: {
+            tooltipContent: formatText({
+              id: 'actionSidebar.tooltip.editColony.colonyName',
+            }),
+          },
+        }}
+        isDisabled={hasNoDecisionMethods}
+      >
+        <FormInputBase
+          message={false}
+          name="colonyName"
+          maxLength={MAX_COLONY_DISPLAY_NAME}
+          placeholder={formatText({
+            id: 'actionSidebar.colonyName.placeholder',
+          })}
+          mode="secondary"
+          disabled={hasNoDecisionMethods}
+        />
+      </ActionFormRow>
+      <ActionFormRow
+        icon="image"
+        fieldName="avatar"
+        title={formatText({ id: 'actionSidebar.colonyLogo' })}
+        tooltips={{
+          label: {
+            tooltipContent: formatText({
+              id: 'actionSidebar.tooltip.editColony.colonyLogo',
+            }),
+          },
+        }}
+        isDisabled={hasNoDecisionMethods}
+      >
+        <div className="flex items-center">
+          <ColonyAvatarField
+            name="avatar"
+            fileOptions={{
+              fileFormat: ['.PNG', '.JPG', '.SVG'],
+              fileDimension: '250x250px',
+              fileSize: '1MB',
+            }}
```

Comment:
This should be some sort of constant as well, as I feel weird having them declared specifically in the component.